packages/core/src/ruleset/rule.ts

```diff
@@ -28,7 +30,7 @@ export interface IRule {
   recommended: boolean;
   documentationUrl: string | null;
   then: IRuleThen[];
-  given: string[];
+  given: GivenDefinition;
```

Comment:
Ah, it used to be array only? Should we keep it that way in that case? Re the `different ways to describe the same thing` comment above.

Reply:
ah shoot, this is the internal representation (that's after all the normalization) we use in spectral, so this indeed only an array. I should revert this particular change